core/src/main/scala/com/databricks/labs/remorph/discovery/SnowflakeTableDefinitions.scala

```diff
@@ -31,7 +31,7 @@ class SnowflakeTableDefinitions(conn: Connection) {
        |    TABLE_CATALOG,
        |    TABLE_SCHEMA,
        |    TABLE_NAME,
-       |    LISTAGG(column_name || ':' ||
+       |    LISTAGG(column_name || '§' ||
```

Comment:
can we move to `~`? This is a common un-conflicting delimiter instead of a Multibyte delimiter?

Reply:
makes sense.